docs/viewlifecycle.md

```diff
@@ -66,8 +66,9 @@ part of the destruction lifecycle.
 |   1   |  `before:destroy` |       `view` - view being destroyed       |
 |       |                   | `...args` - arguments passed to `destroy` |
 |   2*  |  `before:detach`  |       `view` - view being detached        |
-|   3*  |  `detach`         |       `view` - view being detached        |
-|   4   |  `destroy`        |       `view` - view being destroyed       |
+|   3*  |  `dom:remove`     |       `view` - view being detached        |
```

Comment:
I think what is getting confusing is storing the lifecycle events by creation and destruction.  The only reason that `dom:remove` is being triggered here is because the view is detaching.  `dom:refresh` docs say the view that is "rendering".    I'm not sure what it should say in the context of the view destruction..